tests/yam/agama/auto.pm

```diff
@@ -8,13 +8,16 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
+use testapi;
 
 
 sub run {
     assert_screen('agama-main-page', 120);
     assert_screen('agama-installing', 60);
-    assert_screen('welcome-to', 960);
+
+    my $install_product = get_var('INSTALL_PRODUCT', 'Bedrock');
```

Comment:
we are not going to create magic settings in the future, actually we will probably delete most of all of them.
In this case we end up in grub, we could just have an array of arrayref for multiple tags, we don't need to be strict on this test module, we can be more strict and fail in the next module that will be created to login the system or do something on the installed system, in that way we gain as well to not use conditional code in tests.